client/src/components/vote/ResultsModal.tsx

```diff
@@ -15,39 +16,89 @@ export default class ResultsModal extends Component<IResultsModalProps> {
+    }
+
+    dic.sort((first, second) => {
+      return second.result - first.result;
+    });
+
+    return dic;
+  };
+
   public render() {
+    const dic = this.getSortedAnswersWithResults();
+    console.log(dic);
     return (
       <Modal show={this.props.show} onHide={this.props.handleOnHide}>
-        <Modal.Header>
-          <Modal.Title className="text-center">{this.props.voting.info.question}</Modal.Title>
-        </Modal.Header>
-
         <Modal.Body>
           <Panel>
             <Panel.Heading>
-              <Panel.Title componentClass="h3">Voting info</Panel.Title>
+              <Panel.Title className="text-center" style={{ fontSize: "1.5em", fontWeight: "bold" }}>
+                Voting info
+              </Panel.Title>
             </Panel.Heading>
             <Panel.Body>
-              <ControlLabel>Question:</ControlLabel>
+              <ControlLabel style={{ fontSize: "1.2em" }}>Question</ControlLabel>
               <p>{this.props.voting.info.question}</p>
-              <ControlLabel>Possible answers</ControlLabel>
+
+              <ControlLabel style={{ fontSize: "1.2em" }}>Possible answers</ControlLabel>
               <ul>
                 {this.props.voting.info.answers.map((answer) => {
                   return <li>{answer}</li>;
                 })}
               </ul>
+              <ControlLabel style={{ fontSize: "1.2em" }}>Voting deadline</ControlLabel>
+              <p>{covertTimestampToDate(this.props.voting.info.votingEndTime)}</p>
+              <ControlLabel style={{ fontSize: "1.2em" }}>Results expiry date</ControlLabel>
+              <p>{covertTimestampToDate(this.props.voting.info.resultsEndTime)}</p>
+              <ControlLabel style={{ fontSize: "1.2em" }}>Voting type</ControlLabel>
+              <p>{this.props.voting.info.isPrivate ? "Private" : "Public"}</p>
             </Panel.Body>
           </Panel>
 
           <Panel>
             <Panel.Heading>
-              <Panel.Title componentClass="h3">Voting results</Panel.Title>
+              <Panel.Title className="text-center" style={{ fontSize: "1.5em", fontWeight: "bold" }}>
+                Voting results
+              </Panel.Title>
             </Panel.Heading>
             <Panel.Body>
+              <Table responsive>
+                <thead>
+                  <tr>
+                    <th style={{ textAlign: "center" }}>Answer</th>
+                    <th style={{ textAlign: "center" }}>No. votes</th>
+                    <th style={{ textAlign: "center" }}>Percentage</th>
+                  </tr>
+                </thead>
+                <tbody>
+                  {dic.map((element) => (
+                    <tr>
+                      <td style={{ width: "65%" }}>{element.answer}</td>
+                      <td style={{ textAlign: "center" }}>{element.result}</td>
+                      <td style={{ textAlign: "center" }}>{((100 * element.result) / dic.length).toFixed(2)}%</td>
```

Comment:
That's not right

![percentages](https://user-images.githubusercontent.com/22861194/50743029-ec056a80-1212-11e9-9cbd-efed62735ad2.png)

You should divide by sum of all votes (here: 2), not by number of options (6). So it should be like this:
`dic.reduce((partial, obj) => partial + obj.result, 0)`

But I'm not creating a one-click suggestion, because it would be more efficient to evaluate it 1 time (at the top of `render`), not n-times.